internal/servers/controller/controller.go

```diff
@@ -27,6 +30,11 @@ import (
 	ua "go.uber.org/atomic"
 )
 
+const (
+	defaultStatusGracePeriod = 15 * time.Second
```

Comment:
Why did this get changed to 15 from 30? Seems very short.

Reply:
This is to put it in line with the server liveness timeout (seen [here](https://github.com/hashicorp/boundary/blob/33929c27e956385fd3fe72b8f672eb67a9701ffc/internal/servers/repository.go#L16)). This is something I wanted to investigate after finishing the work here. Definitely flexible on changing it, but it made sense to me that if a server was not showing up as "alive" on things like `ListServers`, then it'd probably make sense to start disconnecting things from it at around the same time.